docs/Growing.md

```diff
@@ -72,13 +72,13 @@ and all session state is kept client-side.
 +---------+                        +------------+
 ```
 
-While HTTP methods such as `PUT`, `GET` and `DELETE` are not directly part of REST they go very well with it and are
+While HTTP methods such as `PUT`, `GET` and `DELETE` are not directly part of REST they go well with it and are
```

Comment:
The original says "very". Why did you change it?

Reply:
Because of the quote in the commit message (My text linter complains about the use of very).

Reply:
Then the text should be changed in the original first.

Reply:
I guess this should be ok now that the change has been applied upstream, will resolve this and the other two similar one.